### PR TITLE
chore: update zora video links and descriptions

### PR DESCRIPTION
**Pull request by AshishYelekar**
## Description

Updating zora video links and urls

## Deploy Notes

Notes regarding deployment of the contained body of work. These should note any
db migrations, nginx routes, infrastructure changes, and anything that must be done before deployment.

- [ ] Need to add environment variables
  - `ENV_VAR=`

### CODE REVIEW

File: src/config/partners/zora.ts [hunk 1]
```diff
@@ -68,13 +65,13 @@ const zora: Partner = {
       image: '/partners/zora/drops/FINAL_ART-MalihaAbidi.jpg',
       creator: '0x7be470E6DC93855338a4Cc85D47447811421C7c8',
       name: 'a paragon',
```
Comment (lauchness) [inline src/config/partners/zora.ts:67]: not sure this is intentional
```suggestion
      name: 'A Paragon',
```

**Review by lauchness — APPROVED** [review]
Few questions, but links LGTM

File: src/config/partners/zora.ts (hunk 1 above)
Comment (ximxim) [reply src/config/partners/zora.ts:67]: GTM tracker sheet has it as all lowercase 🤷

File: src/components/ExternalDrop/ExternalDrop.tsx [hunk 2]
```diff
@@ -1,3 +1,5 @@
+'use client'
```
Comment (wilsoncusack) [inline src/components/ExternalDrop/ExternalDrop.tsx:1]: why this change?

**Comment by cb-heimdall** [comment]
Review Error for wilsoncusack @ 2023-08-15 18:37:46 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by lauchness — APPROVED** [review]

**Review by wilsoncusack — APPROVED** [review]

File: src/components/ExternalDrop/ExternalDrop.tsx (hunk 2 above)
Comment (wilsoncusack) [reply src/components/ExternalDrop/ExternalDrop.tsx:1]: ok

**Review by wilsoncusack — APPROVED** [review]

**Review by wilsoncusack — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for lauchness @ 2023-08-15 19:33:54 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by mdehoog — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for wilsoncusack @ 2023-08-15 19:43:07 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by mdehoog — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for mdehoog @ 2023-08-15 19:51:41 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by wilsoncusack — APPROVED** [review]

**Review by mdehoog — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for wilsoncusack @ 2023-08-15 20:01:12 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Comment by cb-heimdall** [comment]
Review Error for wilsoncusack @ 2023-08-15 20:17:52 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by wilsoncusack — APPROVED** [review]

**Review by wilsoncusack — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for mdehoog @ 2023-08-15 20:23:27 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Comment by cb-heimdall** [comment]
Review Error for mdehoog @ 2023-08-15 20:37:24 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)
